components/woocommerce/package.json

```
@@ -0,0 +1,23 @@
+{
+    "name": "@pipedream/woocommerce",
+    "version": "0.0.1",
+    "description": "Pipedream WooCommerce Components",
+    "main": "woocommerce.app.mjs",
+    "keywords": [
+        "pipedream",
+        "woocommerce"
+    ],
+    "homepage": "https://pipedream.com/apps/woocommerce",
+    "author": "Pipedream <support@pipedream.com> (https://pipedream.com/)",
+    "license": "MIT",
+    "dependencies": {
+        "@pipedream/platform": "^0.10.0",
+        "crypto": "^1.0.1",
+        "querystring": "^0.2.1",
+        "woocommerce-api": "^1.5.0"
```

> 2019-07-29: This client is obsolete and will no longer receive updates, a new JavaScript library is available under the name of [@woocommerce/woocommerce-rest-api](https://github.com/woocommerce/woocommerce-rest-api-js-lib).

It seems that this client version is obsolete and was updated to [@woocommerce/woocommerce-rest-api](https://github.com/woocommerce/woocommerce-rest-api-js-lib). They both look like they have the same interface, so switching the dependency probably won't require changing the code, though I'm not sure if it has the `async` methods.

```suggestion
        "@woocommerce/woocommerce-rest-api": "^1.0.1"
```